src/website/src/app/documentation/demos/color/clipboard-copy.service.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Inject, Injectable, PLATFORM_ID } from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
+
+@Injectable()
+export class ClipboardCopyService {
+  private textareaEl: HTMLTextAreaElement;
+
+  constructor(@Inject(PLATFORM_ID) private platformId: Object) {}
+
+  private createTextareaEl() {
+    this.textareaEl = document.createElement('textarea');
+
+    // make it off screen
+    this.textareaEl.setAttribute('readonly', '');
+    this.textareaEl.style.position = 'absolute';
+    this.textareaEl.style.left = '-9999px';
```

Comment:
Is there a better way to force this offscreen? This approach seems heavy handed. What if you just put it at 101% left? Just enough to force it off the screen?

Reply:
ooh I was thinking to use our existing `off-screen` class on that. Thanks for catching it 👍